pyproject.toml

```diff
@@ -121,3 +121,8 @@ checks = [
     "EX01",
     "SA01"
 ]
+exclude = [
+    '\.undocumented_method$',
+    '\.__repr__$',
+    '\.__init__$',
+]
```

Comment:
Please revert this, `__init__` and `__repr__` should be documented.

Reply:
I have reverted it, but currently `__init__` methods are not documented in TopoModelX. This results in failing the tests for any class that you modify.